src/use-image.hook.js

```diff
@@ -56,14 +58,36 @@ export function useImage({processImgElement = noop, fileBlobToUrl = defaultFileB
           fileBlobToUrl(fileInputElement.files[0], imgUrl => {
             performCommandWithValue(imgUrl)
             const imgElement = document.querySelector(`img[src="${imgUrl}"]`)
+            if (imgElement.src && imgElement.src.startsWith('blob:')) {
```

Comment:
This won't be true for Canopy or any user who provides their own `fileBlobToUrl` function

Reply:
Although I haven't formed a cohesive thought on this yet, I have some concerns about DataURL's since I believe they take on the Content Security Policy of the executing context.